public/video-ui/src/components/FormFields/SelectBox.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+export default class SelectBox extends React.Component {
+
+  renderField = () => {
+    if(this.props.editable) {
```

Comment:
To remove nesting can we just return within `if (!this.props.editable)` then put this code at the bottom of the function